tracing-attributes/src/lib.rs

```diff
@@ -168,6 +168,30 @@ use syn::{
 ///     # Ok(())
 /// }
 /// ```
+///
+/// It also works with [async-trait](https://crates.io/crates/async-trait) (a crate that allows
+/// async functions on traits, something not currently possible in rust), and hopefully most
+/// libraries that exhibit similar behaviors:
```

Comment:
Nitpick: can we wrap these comments consistently with other comments in this file?